### PR TITLE
Added AppCenter SDK for analytics, diagnostics, and more!

### PR DESCRIPTION
**Pull request by IsaacHardy**
This simply setups up the client app to use Appcenter's SDK for analytics and diagnostics. No further work was added besides one simply event capture. 

## Creator

- [ ] Build relevant tests if any apply
- [ ] Ensure there are no new warnings (in app AND in storybook)
- [ ] Upload an animated GIF showcasing the solution (if it applies)
- [ ] Set a relevant reviewer
- [ ] PR has a relevant title that adheres to our patch-notes/change-log style

## Reviewer

- [ ] Review CI build results
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review new stories if any apply
- [ ] Review test coverage: Are all new or changed states tested?
- [ ] Review new test logic
- [ ] Review the change-log entry
- [ ] Review that the PR title adheres to our patch-notes/change-log style

### CODE REVIEW

File: packages/apolloschurchapp/src/content-feed/index.js [hunk 1]
```diff
@@ -34,11 +34,14 @@ class ContentFeed extends PureComponent {
   /** Function that is called when a card in the feed is pressed.
    * Takes the user to the ContentSingle
    */
-  handleOnPress = (item) =>
+  handleOnPress = (item) => {
+    Analytics.trackEvent('Content Single', { itemId: item.id });
```
Comment (IsaacHardy) [inline packages/apolloschurchapp/src/content-feed/index.js]: Here is the simple event tracker.

Comment (conrad-vanl) [reply packages/apolloschurchapp/src/content-feed/index.js]: Two things: 

1. This probably isn't the right place to track this specific event
2. Can we abstract this over to our `util/analytics` code like we did on Newspring/Apollos?

Comment (vinnyjth) [reply packages/apolloschurchapp/src/content-feed/index.js]: @conrad-vanl @IsaacHardy I propose we hold off on implementing any Analytics, I'm working on analytics pr that will open in the next couple of days that will replace this implementation :) (but have Appcenter as a possible backend)

**Comment by vinnyjth** [comment]
@IsaacHardy I'm running into issues building this PR. I tried `pod install` but that created a large number of files and left me unable to build. I reset back to your branch and built, but ran into this issue:
https://www.dropbox.com/s/h8pl8lhwedcvtbx/Screenshot%202018-08-31%2017.09.55%287%29.png?dl=0

Do you have a set of steps to run to get the project setup with cocoa pods?

File: packages/apolloschurchapp/src/content-feed/index.js (hunk 1 above)
Comment (IsaacHardy) [reply packages/apolloschurchapp/src/content-feed/index.js]: @vinnyjth @conrad-vanl This PR was simply created to help track "some" work I did before @vinnyjth took over analytics. The even tracker is only used as a test to make sure the implementation was complete. 

I have no issues of rendering this PR void.

**Comment by conrad-vanl** [comment]
`cocoa pods` on a react-native project 😭 why is it always so hard

rubygems 👎👎👎👎👎👎👎!!!! 😆

**Comment by conrad-vanl** [comment]
Removing the ready for review label

**Comment by vinnyjth** [comment]
No plans to merge this right now. We would love to have the bug tracking AppCenter, but the Cocoa Pods dependency is adding trouble :/ Will revisit sometime soon :D Thanks for the excellent work @IsaacHardy and I think we can leave this open.

**Comment by IsaacHardy** [comment]
@vinnyjth Do you still need this PR or can we close and delete the branch?

**Comment by vinnyjth** [comment]
@IsaacHardy We can keep this open, actually. @conrad-vanl and I were discussing the possibility of ditching OneSignal and the analytics interface and going all in on AppCenter, in which case this PR would be great to have :)

**Comment by IsaacHardy** [comment]
@vinnyjth Because that there have been a bunch of changes to the code since creating this PR, I recommend we use this only as a documentation of how we would implement AppCenter into the app.

**Comment by conrad-vanl** [comment]
Closing this due to stale-ness 🍞

**Pull request closed without merging**